README.md

```diff
@@ -91,6 +102,9 @@ pip install abdm-python-integrator@git+https://github.com/dimagi/abdm-python-int
     url(r'^abdm/', include('abdm_integrator.urls')),
     ```
 
+> NOTE: If you are using the default celery settings for  `ABDM_INTEGRATOR`, add desired celery configurations in django  settings with `CELERY_` prefix.\
+ Sample command to run Celery ` celery -A abdm_integrator.celery_app worker -l info`
```

Comment:
How are you planning to do this in production when adding it into CommcareHQ?

Reply:
(We could use the existing ones or can create new ones)
Wanted to discuss on this and get your inputs.  Will sync offline.

Reply:
@sravfeyn .  Thanks for the inputs.
Uses [shared task](https://docs.celeryq.dev/en/stable/userguide/tasks.html#:~:text=If%20you%E2%80%99re%20using%20Django%20(see%20First%20steps%20with%20Django)%2C%20or%20you%E2%80%99re%20the%20author%20of%20a%20library%20then%20you%20probably%20want%20to%20use%20the%20shared_task()%20decorator%3A) by default now, This does not require celery app and uses the default one configured in the project.
The setting `CELERY_APP` is included so that it be configured in HQ to use the celery app (wrapper).